clients/api/api.py

```diff
@@ -0,0 +1,10 @@
+from clients.dbt.dbt_runner import DbtRunner
+
+
+class APIClient:
```

Comment:
In the future we will add a wrapper that adds error/successful status for each call 🙂